add diamond post (#259)
usegalaxy-eu/website: _posts/2019-04-23-diamond.md

---
site: freiburg

> title: "Visit from Macromolecular Crystallography Group"

Reviewer: ```suggestion
title: "Visit from Diamond's Macromolecular Crystallography Group"
```

tags: [tools]
supporters:
 - eosc
---

> Recently, we have started working with the (Macromolecular Crystallography group)[https://www.diamond.ac.uk/Instruments/Mx.html], which is based at the [Diamond synchrotron](https://www.diamond.ac.uk) in Didcot, UK. We have a common interest in enabling reproducible, user-friendly data analysis in cheminformatics and computational chemistry. They are developing (Squonk)[https://squonk.it], a computational notebook with similar aims to the [ChemicalToolbox](http://cheminformatics.usegalaxy.eu) we are working on.

Reviewer: ```suggestion
Recently, we have started working with the (Macromolecular Crystallography group)[https://www.diamond.ac.uk/Instruments/Mx.html], which is based at the [Diamond synchrotron](https://www.diamond.ac.uk) in Didcot, UK. We have a common interest in enabling reproducible, user-friendly data analysis in cheminformatics and computational chemistry. As part of their [XChem project](https://www.diamond.ac.uk/Instruments/Mx/Fragment-Screening.html) they are developing computational workflows to assist in designing follow-up compounds from fragment screening hits.

Diamond is working with [Informatics Matters](https://www.informaticsmatters.com/) to this aim, and their (Squonk Computational Notebook)[https://squonk.it], a computational notebook with similar aims to the [ChemicalToolbox](http://cheminformatics.usegalaxy.eu) we are working on, is one part of the end user facing tools. We were interested in investigating how Squonk and Galaxy could be integrated. 
```

> Last week, [Tim Dudgeon](https://www.informaticsmatters.com) and [Rachael Skyner](https://www.diamond.ac.uk/Instruments/Mx/Fragment-Screening/XChem-team/Skyner.html) visited Freiburg to find out more about Galaxy. We spent two days discussing and together developed a new Galaxy tool which can be used as part of a protein-ligand docking workflow. Eventually, we hope to create a common base of tools for compuational chemistry that can be utilized by both of our platforms.

Reviewer: ```suggestion
Recently, [Tim Dudgeon](https://www.linkedin.com/in/timdudgeon/) from Informatics Matters and [Rachael Skyner](https://www.diamond.ac.uk/Instruments/Mx/Fragment-Screening/XChem-team/Skyner.html) from Diamond visited Freiburg to find out more about Galaxy. We spent two days discussing and together developed a new Galaxy tool which can be used as part of a protein-ligand docking workflow. The aim is to get a Galaxy instance running at Diamond in the near future, and eventually, we hope to create a common base of tools for compuational chemistry that can be utilized by both of our platforms.
```
